docs/semgrep-appsec-platform/notifications.md

```diff
@@ -4,25 +4,66 @@ append_help_link: true
 title: Alerts and notifications
 hide_title: true
 toc_max_heading_level: 4
-description: "Learn how to receive Slack or email alerts about findings and failures and how to integrate using webhooks."
+description: "Learn how to receive Slack, webhook, or email notifications about findings."
 tags:
     - Deployment
     - Semgrep AppSec Platform
 ---
 
+import RuleModes from "/src/components/reference/_rule-modes.md"
+
 # Alerts and notifications
 
-Users can receive notifications from Semgrep AppSec Platform in the following channels:
+You can receive notifications for Semgrep findings in the following channels:
+
+<table>
+<thead>
+<tr>
+<th>Channel</th>
+<th>Semgrep Code</th>
+<th>Semgrep Supply Chain</th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td>[Slack](/semgrep-appsec-platform/slack-notifications)</td>
+<td rowspan="3">Integrate with Semgrep through **Settings > Integrations**. Customize through rule modes in [Policies page](/semgrep-code/policies).</td>
+<td>Integrate with Semgrep through **Settings > Integrations**. Limited customizability; configured by default to send notifications on <strong>reachable</strong> findings</td>
```

Comment:
Is this currently accurate? Might be worth double-checking - I know policies only cover block/comment, but I recall there being other conditions on notifications before.

Reply:
For SSC, it is reachable findings (including **Always reachable**, i.e. maldeps). I tested this recently with Derian. There is no visible "you're getting reachable findings" anywhere AFAIK, not in Policies > Supply Chain (see screenshot); the only place where we indicate that you can get Supply Chain findings is in the notification in the workspace Slack you've installed Semgrep (see screenshot) under **Select target scan type**

![Snag_14614af](https://github.com/user-attachments/assets/cea0f7e5-fb51-4eb6-a3a1-ac1d5aab1a33)

![Snag_1462f0d](https://github.com/user-attachments/assets/be876de1-f21c-42c0-a73f-cbda53f5c805)

Reply:
cc @armchairlinguist ?

Reply:
Oh right, sorry - I did the thing again where I was thinking it was only critical and high, but it's not - it's always been just reachable, only most of our reachables for a long time were critical/high.

Reply:
All good. I'll go ahead with the merge then! 👍